mimic/rest/nova_api.py

```diff
@@ -328,10 +328,10 @@ def get_flavor_details(self, request, tenant_id, flavor_id):
         """
         Returns a get flavor response, for any given flavorid
         """
-        return (
-            self._region_collection_for_tenant(tenant_id)
-            .get_flavor(request, flavor_id, absolutize_url=self.url)
-        )
+        flavor_collection = GlobalFlavorCollections(tenant_id=tenant_id,
+                                                    clock=self._session_store.clock)
```

Comment:
My gut feel here is that the class should be called `GlobalFlavorCollection` (strike the "s" at the end). Reading this as "flavor_collection is a new global flavor collections" and it feels off.